packages/material-ui/src/styles/muiStyled.d.ts

```diff
@@ -0,0 +1,208 @@
+import * as React from 'react';
+import * as CSS from 'csstype';
+
+export interface SerializedStyles {
```

Comment:
The styles has been copied from emotion's codebase, and adjusted to the changes that we have.